client/src/main.rs

```diff
@@ -353,134 +359,40 @@ fn main() {
-                        let demurrage_rate = ln::<BalanceType, BalanceType>(BalanceType::from_num(0.5)).unwrap()
-                            .checked_mul(BalanceType::from_num(-1)).unwrap()
-                            .checked_div(BalanceType::from_num(demurrage_halving_blocks)).unwrap();
-                        info!("demurrage halving blocks: {} which translates to a rate of {} ",
-                            demurrage_halving_blocks, hex::encode(demurrage_rate.encode()));
-                        maybe_demurrage = Some(demurrage_rate);
-                    } else {
-                        info!("using default demurrage");
-                    }
-
-                    let mut maybe_income: Option<BalanceType> = None;
-                    if let Ok(income) = serde_json::from_value::<f64>(spec["community"]["ceremony_income"].clone()) {
-                        info!("ceremony income specified as {}", income);
-                        maybe_income = Some(BalanceType::from_num(income));
-                    } else {
-                        info!("using default income");
-                    }
-
-                        let sudoer = AccountKeyring::Alice.pair();
+                    let sudoer = AccountKeyring::Alice.pair();
                     let api = get_chain_api(matches).set_signer(sudoer);
 
-                    let call = compose_call!(
-                        api.metadata.clone(),
-                        "EncointerCommunities",
-                        "new_community",
-                        loc[0],
-                        bootstrappers,
-                        meta,
-                        maybe_demurrage,
-                        maybe_income
-                    );
-                    let unsigned_sudo_call = compose_call!(
-                        api.metadata.clone(),
-                        "Sudo",
-                        "sudo",
-                        call.clone()
-                    );
-                    info!("raw sudo call to sign with js/apps {}: 0x{}", cid, hex::encode(unsigned_sudo_call.encode()));
-                    let xt: UncheckedExtrinsicV4<_> =
-                        compose_extrinsic!(api.clone(), "Sudo", "sudo", call);
-                    ensure_payment(&api, &xt.hex_encode());
-                    let tx_hash = api.send_extrinsic(xt.hex_encode(), XtStatus::InBlock).unwrap();
-                    info!("[+] Transaction got included. Hash: {:?}\n", tx_hash);
-                    println!("{}", cid);
-
                     if api.get_current_phase().unwrap() != CeremonyPhaseType::REGISTERING {
-                        error!("wrong ceremony phase for registering new locations for {}", cid);
+                        error!("Wrong ceremony phase for registering new locations for {}", cid);
+                        error!("Exiting process without registering a new community");
                         std::process::exit(exit_code::WRONG_PHASE);
```

Comment:
Before this was called after registering the community. Was there a reason for that?

If not, I thought it is more intuitive to not register the community at all.

Reply:
Registering a new community is always allowed. Registering new locations only during REGISTERING phase. I'd leave it like that because it's handy. Also, we may still want to call this function just to dump the encoded call data to submit with different origin via js/apps

Reply:
Ok, I changed that back to how it was before.